.github/workflows/build_release.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and push a release
+on:
+  release:
+    types: [created]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```

Comment:
Should we use one of the latest tagged versions of the login-action? This commit is from 2023 March where the latest tagged v3 is from September.

Reply:
Using now the latest tag for all the actions